dev-java/tomcat-native/tomcat-native-1.1.33-r1.ebuild

```diff
@@ -13,30 +13,42 @@ SRC_URI="mirror://apache/tomcat/tomcat-connectors/native/${PV}/source/${P}-src.t
 HOMEPAGE="http://tomcat.apache.org/"
 KEYWORDS="~amd64 ~x86"
 LICENSE="Apache-2.0"
-IUSE=""
+IUSE="test"
 
-RDEPEND="=dev-libs/apr-1*
+RDEPEND="dev-libs/apr:1
 	dev-libs/openssl:=
-	>=virtual/jre-1.5:*"
+	>=virtual/jre-1.7
+	test? ( dev-java/ant-junit:0 )"
 
-DEPEND=">=virtual/jdk-1.5:*
+DEPEND=">=virtual/jdk-1.7
 	${RDEPEND}"
 
-S=${WORKDIR}/${P}-src/jni/native
+S=${WORKDIR}/${P}-src/jni
 
 src_configure(){
+	cd native
 	econf --with-apr=/usr/bin/apr-1-config  \
 		--with-ssl=/usr || die "Could not configure native sources"
 }
 
 src_compile() {
+	eant jar -f build.xml
+
+	cd native
 	base_src_compile
 }
 
 src_install() {
+	java-pkg_newjar dist/${PN}-${PV}-dev.jar ${PN}.jar
+
+	cd native
```

Comment:
https://github.com/gentoo/gentoo/commit/6d93b2e2efe4034288ceb9f61f932bef44cb1af1 addresses your remarks. Thank you!